src/test/python/entries.py

```diff
@@ -0,0 +1,134 @@
+import random
+from datetime import datetime, timedelta
+
+import mysql.connector
+import requests
+
+mydb = mysql.connector.connect(
+    host="localhost",
+    user="root",
+    password="password",
+    database="wemove"
+)
+
+myCursor = mydb.cursor()
+
+
+def generate_consistent_entries(userid, start_date, subDuration):
+    date = start_date
+    end_date = start_date + timedelta(days=subDuration)
+
+    if subDuration == 0:
+        randomHour = random.randint(8, 20)
+        entry_time = datetime(date.year, date.month, date.day, randomHour, 0)
+        leave_time = datetime(date.year, date.month, date.day, randomHour + 2, 0)
+        myCursor.execute(insert_INTO_entry, (entry_time, leave_time, userid))
+        print(entry_time, leave_time, userid)
+        return
+
+    while date <= end_date:
+        if date.weekday() < 5:
```

Comment:
gym is open all week . 
Members with regular job usually come 3 days in the week after work.
there is less people on week-end usually